evals/mmmu/utils.py

```diff
@@ -1,7 +1,7 @@
 import ast
 import os
 
-from datasets import concatenate_datasets, load_dataset  # type: ignore
```

Comment:
The reason this is here is that we get this "problem" in VS Code without the `# type: ignore`:

<img width="931" alt="Screenshot 2024-09-23 at 21 09 23" src="https://github.com/user-attachments/assets/fff79daf-2912-4c0b-b6f5-b15cab8140f8">

I'm wondering if you aren't getting this b/c perhaps you are using pyright rather than mypy? (that's on my list to investigate as well). I was also under the impression that the datasets package doesn't have any type declarations available (so we end up w/ this everywhere we use datasets -- I would _love_ to remove this though, LMK if you have any info on making this better).

Reply:
Sounds good, will let you know when the change to make `evals` into a proper package lands.